### DISCUSSION TITLE
Klipper firmware update to SV06PLUS

### DISCUSSION THREAD

**Discussion by SV06PLUSNOOB** [bassamanator/Sovol-SV06-firmware#38] [General]
I put the klipper firmware on my 8gb sdcard. I dont see the moving printhead on the little screen during start up. The screen just stays blue with SV06PLUS on the middle. Does this mean its been flashed to Klipper ?

**Comment by fmarulaz** [comment]
Certainly not.
You must flash another time. (And rename firmware with a different name !)

Bien cordialement :
Florent MARULAZ

**Comment by fmarulaz** [comment]
Are you working on a Mac ?
Is your sdcard formatted with 4096 sector’s Fat32 ?
Did you use USB 3 on your RPI ?

Bien cordialement :
Florent MARULAZ

**Reply by bassamanator** [reply]
Read the documentation.

**Reply by bassamanator** [reply]
Read the flashing instructions.

**Reply by bassamanator** [reply]
Great. Please close this discussion when ready.

**Comment by SV06PLUSNOOB** [comment]
For who who has these problems. Format the card on a windows computer. After flashing You just get a blue screen on the printer that says sv06 plus. You see nothing happing during flashing.

To find the comm port 
sudo ls -l /dev/serial/by-id/*

For me it gave 
/dev/serial/by-id/usb-1a86_USB_Serial-if00-port0 -> ../../ttyUSB0

This is in my printer.cfg file.
[mcu]
serial: /dev/ttyUSB0
restart_method: command

You can just edit the file by clicking on it in MAINSAIL and change or add lines.
You can paste everything you want in the file.

Only doubt i have is the cable. I do loose connection even after using klipper. I ordered a new one.

**Comment by SV06PLUSNOOB** [comment]
Thx for the explanation.

**Reply by bassamanator** [reply]
Welcome.

I'm not sure if your setup is as it should be because you should not have received that `PAUSE` macro missing error. I recommend you go over the setup again, but it's up to you.

**Reply by SV06PLUSNOOB** [reply]
Thx tomorrow new cable will arrive. Any things is need to change because i have the PLUS version. It says SV06 on the rep.

**Reply by bassamanator** [reply]
There is a separate branch (page) for the [sv06-plus](https://github.com/bassamanator/Sovol-SV06-firmware/tree/sv06-plus).
